packages/fxa-settings/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactNode } from 'react';
+import { useClickOutsideEffect } from '@fxa-components/lib/hooks';
+import classNames from 'classnames';
+import Portal from '@fxa-components/Portal';
+import { ReactComponent as CloseIcon } from '@fxa-components/images/close.svg';
+
+type ModalProps = {
+  className?: string;
+  onDismiss?: Function;
+  onConfirm?: () => void;
+  children: ReactNode;
+  'data-testid'?: string;
```

Comment:
Does this need to be optional if it's assigned a default value in `Modal` instantiation?

Reply:
Yeah, I just removed it to double check and I get complaints.